src/main/java/com/thanksd/server/service/DiaryService.java

```diff
@@ -0,0 +1,96 @@
+package com.thanksd.server.service;
+
+import com.thanksd.server.domain.Diary;
+import com.thanksd.server.domain.Member;
+import com.thanksd.server.dto.request.DiaryRequest;
+import com.thanksd.server.dto.response.diary.DiaryAllResponse;
+import com.thanksd.server.dto.response.diary.DiaryIdResponse;
+import com.thanksd.server.dto.response.diary.DiaryResponse;
+import com.thanksd.server.exception.notfound.NotFoundDiaryException;
+import com.thanksd.server.exception.notfound.NotFoundMemberException;
+import com.thanksd.server.repository.DiaryRepository;
+import com.thanksd.server.repository.MemberRepository;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@Transactional(readOnly = true)
+@RequiredArgsConstructor
+public class DiaryService {
+
+    private final DiaryRepository diaryRepository;
+    private final MemberRepository memberRepository;
+    private final MemberService memberService;
+
+    @Transactional
+    public DiaryIdResponse saveDiary(DiaryRequest diaryRequest, Long memberId) {
+        Member member = memberRepository.findById(memberId)
+                .orElseThrow(() -> new NotFoundMemberException());
```

Comment:
이 부분에서 지금 `Lambda can be replaced with method reference`뜨는데
```suggestion
               .orElseThrow(NotFoundMemberException::new);
```

이렇게 개선할 수 있는데 어떠신가요??